specification/translation/Azure.AI.TextTranslation/models-translate.tsp

```diff
@@ -151,28 +151,30 @@ union TextType {
   Html: "Html",
 }
 
+#suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "Enums should be defined without the `@fixed` decorator."
+#suppress "@azure-tools/typespec-azure-core/no-enum" "Azure services should not use the enum keyword."
+@fixed
 @doc("Translator profanity actions")
-union ProfanityAction {
-  string,
-
-  /** Take no action. */
+enum ProfanityAction {
+  @doc("No Action is taken on profanity")
   NoAction: "NoAction",
 
-  /** Replace the profanity with a specific marker. */
+  @doc("Profanity is marked.")
   Marked: "Marked",
 
-  /** Delete the profanity. */
+  @doc("Profanity is deleted from the translated text.")
   Deleted: "Deleted",
 }
 
+#suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "Enums should be defined without the `@fixed` decorator."
```

Comment:
Why is this being suppressed?

Reply:
It came after the discussion in TypeSpec channel that if we want to use enums, we need to suppress. I will tag you un the conversation.